### DISCUSSION TITLE
[ERROR] /usr/local/nebula/etc/nebula-metad.conf not found

### DISCUSSION THREAD

**Discussion by porscheme** [vesoft-inc/nebula#4660] [Q&A]
Nebula version: v3.2.0

Cluster is working fine but nebula.service script doesn't work as expected
Please provide proper documentation on how to use this script.
We followed this [documentation](https://docs.nebula-graph.io/3.2.0/2.quick-start/5.start-stop-service) to test

We are getting below error when trying to check the status of Nebula services. Does anyone know how to troubleshoot?
```
[root@nebula-cluster-storaged-0 nebula]# sh scripts/nebula.service -v status all
/usr/local/nebula/scripts/utils.sh: line 159: /usr/local/nebula/bin/nebula-metad: No such file or directory
☺☻[ERROR]☺☻ /usr/local/nebula/etc/nebula-metad.conf not found
```

**Step to reproduce**
- List all the pods of Nebula Service by running below command
```
kubectl get pods <Nebula_K8S_Namespace>
```
- Log on to any storaged pod by running below command
```
kubectl exec -it <Any_Nebula_Storage_Pod> -n <Nebula_K8S_Namespace> -- bash
```

- Run nebula.service script to repro the error
```
cd /usr/local/nebula
./scripts/nebula.service -v status all
```

- List files in  /usr/local/nebula/etc directory and notice only nebula-storaged.conf file was present
```
[root@nebula-cluster-storaged-0 nebula]# cd /usr/local/nebula/etc/
[root@nebula-cluster-storaged-0 etc]# ls
nebula-storaged.conf
```

**Comment by wey-gu** [comment]
> Nebula version: v3.2.0
> 
> 
> 
> Cluster is working fine but nebula.service script doesn't work as expected
> 
> Please provide proper documentation on how to use this script.
> 
> We followed this [documentation](https://docs.nebula-graph.io/3.2.0/2.quick-start/5.start-stop-service) to test
> 
> 
> 
> We are getting below error when trying to check the status of Nebula services. Does anyone know how to troubleshoot?
> 
> ```
> 
> [root@nebula-cluster-storaged-0 nebula]# sh scripts/nebula.service -v status all
> 
> /usr/local/nebula/scripts/utils.sh: line 159: /usr/local/nebula/bin/nebula-metad: No such file or directory
> 
> ☺☻[ERROR]☺☻ /usr/local/nebula/etc/nebula-metad.conf not found
> 
> ```
> 
> 
> 
> **Step to reproduce**
> 
> - List all the pods of Nebula Service by running below command
> 
> ```
> 
> kubectl get pods <Nebula_K8S_Namespace>
> 
> ```
> 
> - Log on to any storaged pod by running below command
> 
> ```
> 
> kubectl exec -it <Any_Nebula_Storage_Pod> -n <Nebula_K8S_Namespace> -- bash
> 
> ```
> 
> 
> 
> - Run nebula.service script to repro the error
> 
> ```
> 
> cd /usr/local/nebula
> 
> ./scripts/nebula.service -v status all
> 
> ```
> 
> 
> 
> - List files in  /usr/local/nebula/etc directory and notice only nebula-storaged.conf file was present
> 
> ```
> 
> [root@nebula-cluster-storaged-0 nebula]# cd /usr/local/nebula/etc/
> 
> [root@nebula-cluster-storaged-0 etc]# ls
> 
> nebula-storaged.conf
> 
> ```
> 
> 
> 
> 
> 
> 

This script is for deb/rpm package, not for container images :)

The process should be start by container entrypoint instead in operators/k8s deployment of nebulagraph
